great-idea/services.html

```diff
@@ -15,93 +15,109 @@
 </head>
 
 <body>
-
-
-Services
-Product
-Vision
-Features
-About
-Contact
-
-<img class="logo" src="img/logo.png" alt="Great Idea! Company logo.">
-
+  <div class="container2">
+    <div class="header2">
+       <nav class="nav2">   
+          <a class="a2" href='services.html'>Services</a>
+          <a class="a2" href="#">Product</a>
+          <a class="a2" href="#">Vision</a>
+          <a class="a2" href="#">Features</a>
+          <a class="a2" href="#">About</a>
+          <a class="a2" href="#">Contact</a>
+          <img class="logo2" src="img/logo.png" alt="Great Idea! Company logo.">
```

Comment:
Always have a way for users to get back the the home page, adding a href link on the logo img would have been a good way to accomplish this!